scripts/mnist_experiment.py

```
@@ -1,62 +1,425 @@
+            experiment_config=experiment_config,
+            mode="create",
+            online=False,
+            project_name="cdt-plusplus",
+        )
+
+    from comet_ml.integration.pytorch import log_model, watch  # noqa: PLC0415
+
+    return _CometRun(experiment=experiment, log_model=log_model, watch=watch)
+
+
+def _build_model(torch_module: Any) -> Any:
```

Ruff (0.16.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `torch_module`

(ANN401)